config/config.exs

```diff
@@ -8,6 +8,14 @@ if Mix.env() == :test do
     report_dir: "reports/exunit"
 
   config :opencensus,
-    reporters: [{Opencensus.TestSupport.SpanCaptureReporter, []}],
-    send_interval_ms: 100
+    reporters: [
+      {Opencensus.TestSupport.SpanCaptureReporter, []}
+    ],
+    send_interval_ms: 100,
+    stat: [
+      export_interval: 100,
+      exporters: [
+        {Opencensus.TestSupport.MetricsCaptureExporter, []}
```

Comment:
I would dynamically attach exporter in tests as it will allow to make test async.

Reply:
Hmm, I don't think attaching it dynamically makes any difference. I think it's fine to just make tests async (which I just did)